### ISSUE TITLE
Batch size and training resource

### ISSUE THREAD

**Issue by unilight** [yukara-ikemiya/wavefit-pytorch#2]
Hi, thanks for the nice work!

I am trying to run training using the default setting.
I think your original setting used a batch size of 64 per GPU, and I think you used 8 GPUs.
I don't have 8 GPUS so I am trying to train using just one GPU (Tesla V100, 32GB), and I reduced the batch size to 64.
However I ran into OOM.

I am wondering what kind of GPU you used?
And I am also wondering whether you experimented with different batch sizes, and how the performance changes?

Looking forward to your response!

**Comment by yukara-ikemiya** [comment]
The batchsize `512` in the sample job code is referenced from the original setting Google used for their training in the paper. And they used 128 TPUs for the training (4 samples per TPU).
<img width="662" height="190" alt="Image" src="https://github.com/user-attachments/assets/107b0828-212a-4929-8525-1b95c9ef66e5" />

Unfortunately, since I also don't have such large GPU/TPU cluster, I conducted only small experiments with smaller batch sizes.
When using a small batch size, I think it's necessary to adjust the learning rate -- typically lowering it -- to stabilize training. However, I believe it's possible to achieve performance comparable to that with larger batch sizes. 
That said, since smaller batches require more training iterations, the overall training time tends to be significantly longer..
